spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/MicrosoftTeamsNotifier.java

```diff
@@ -131,12 +143,12 @@ private String safeFormat(String format, Object... args) {
 		}
 	}
 
-	protected Message createMessage(Application app, String registeredTitle,
+	protected HttpEntity<String> createMessage(Application app, String registeredTitle,
```

Comment:
Just pass along the Message object and let Spring's `MessageConverter` handle the serialization

Reply:
Yep, done.